common/src/main/java/com/wynntils/features/user/AddCommandExpansionFeature.java

```diff
@@ -0,0 +1,267 @@
+                .then(literal("invite").then(argument("player", StringArgumentType.string())))
+                .then(literal("join").then(argument("tag", StringArgumentType.greedyString())))
+                .then(literal("kick").then(argument("player", StringArgumentType.string())))
+                .then(literal("leaderboard"))
+                .then(literal("leave"))
+                .then(literal("list"))
+                .then(literal("log"))
+                .then(literal("manage"))
+                .then(literal("rank")
+                        .then(argument("player", StringArgumentType.string()))
+                        .then(argument("rank", StringArgumentType.string())))
+                .then(literal("rewards"))
+                .then(literal("stats"))
+                .then(literal("territory"))
+                .then(literal("xp").then(argument("amount", IntegerArgumentType.integer())))
+                .build();
+        root.addChild(node);
+
+        root.addChild(literal("gu").redirect(node).build());
+    }
+
+    private void addIgnoreCommandNode(RootCommandNode root) {
+        root.addChild(literal("ignore")
+                .then(literal("add").then(argument("player", StringArgumentType.string())))
+                .then(literal("remove").then(argument("player", StringArgumentType.string())))
+                .build());
+    }
+
+    private void addHousingCommandNode(RootCommandNode root) {
+        CommandNode<CommandSourceStack> node = literal("housing")
+                .then(literal("allowedit").then(argument("player", StringArgumentType.string())))
+                .then(literal("ban").then(argument("player", StringArgumentType.string())))
+                .then(literal("disallowedit").then(argument("player", StringArgumentType.string())))
+                .then(literal("edit"))
+                .then(literal("invite").then(argument("player", StringArgumentType.string())))
+                .then(literal("kick").then(argument("player", StringArgumentType.string())))
+                .then(literal("kickall"))
+                .then(literal("leave"))
+                .then(literal("public"))
+                .then(literal("unban").then(argument("player", StringArgumentType.string())))
+                .then(literal("visit"))
+                .build();
+        root.addChild(node);
+
+        root.addChild(literal("is").redirect(node).build());
+    }
+
+    private void addMessagingCommandNodes(RootCommandNode root) {
+        root.addChild(literal("g")
+                .then(argument("msg", StringArgumentType.greedyString()))
+                .build());
+
+        root.addChild(literal("p")
+                .then(argument("msg", StringArgumentType.greedyString()))
+                .build());
+
+        root.addChild(literal("r")
+                .then(argument("msg", StringArgumentType.greedyString()))
+                .build());
+
+        CommandNode<CommandSourceStack> node4 = literal("msg")
```

Comment:
Why does this one have a `4` behind the variable name?

Reply:
Debug code left behind...